packages/roosterjs-content-model/lib/publicApi/block/toggleMargins.ts

```diff
@@ -0,0 +1,34 @@
+import { getObjectKeys } from 'roosterjs-editor-dom';
+import { createParagraphDecorator } from '../../modelApi/creators/createParagraphDecorator';
+import { MarginFormat } from '../../publicTypes/format/formatParts/MarginFormat';
+import { IContentModelEditor } from '../../publicTypes/IContentModelEditor';
+import { formatParagraphWithContentModel } from '../utils/formatParagraphWithContentModel';
+
+const tagNamesSupportingMargins = 'p,h1,h2,h3,h4,h5,h6';
+
+/**
+ * Toggles the current block(s) margin properties.
+ * @param editor The editor to operate on
+ * @param marginFormat Object containing margin props to be changed.
+ * If a margin property is not in this object, it won't be changed.
+ */
+export default function toggleMargins(
+    editor: IContentModelEditor,
+    marginFormat: Pick<MarginFormat, 'marginTop' | 'marginBottom'>
```

Comment:
I think the easiest way to organize the parameters is to have 2 separate parameters for marginTop and margin Bottom.

Both of the are allowed to be undefined to keep current value.